modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc

```diff
@@ -3,9 +3,16 @@
 [id="selecting-an-open-vsx-registry-instance"]
 = Selecting an Open VSX registry instance
 
-The Open VSX registry at `pass:c,a,q[https://open-vsx.org]` is the default if resolved from within your organization's cluster. If not, then the embedded Open VSX registry within the {prod-short} `plugin-registry` pod is the default.
+The embedded Open VSX registry within the {prod-short} `plugin-registry` is the default.
```

Comment:
the embedded Open VSX registry is default only for DevSpaces, for Che the default one is https://open-vsx.org

Reply:
@svor and @ibuziuk, is the following still correct?
```
The Open VSX registry at `pass:c,a,q[https://open-vsx.org]` is the default if resolved from within your organization's cluster.
```
I mean does Che/DS check whether https://open-vsx.org is resolved or not? I thought it was an automated decision tree: if https://open-vsx.org is not resolved (that means Che/DS is in an airgapped environment), then Che/DS switches to using the embedded registry. And now DS does not check whether https://open-vsx.org is resolved, correct? Does Che still check whether https://open-vsx.org is resolved?

Reply:
as far as I know we don't check if open-vsx.org is resolved